app-text/cutemarked/cutemarked-0.11.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="A Qt-based Markdown editor"
+DESCRIPTION="CuteMarkEd is a Qt-based Markdown editor with live HTML preview, \
+math expressions, code and markdown syntax highlighting."
+HOMEPAGE="https://cloose.github.io/CuteMarkEd"
+COPYRIGHT="2013 Christian Loose
+	2007 David Loren Parsons
+	2011-2012 Ali Rantakari
+	2012 CodeCatalyst, LLC"
+LICENSE="GNU GPL v2
+	MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/cloose/CuteMarkEd/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="78a41808c2f0452375810abdff76eeaaee012f8d1368a2b8772ec6b4d2ceeec8"
+SOURCE_DIR="CuteMarkEd-$portVersion"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="?x86"
```

Comment:
Thanks @Begasus for your test and [message](https://github.com/haikuports/haikuports/pull/2057#issuecomment-355730628)!

@TypicalGitHubUser, this means we can change this:
```diff
-SECONDARY_ARCHITECTURES="?x86"
+SECONDARY_ARCHITECTURES="x86"
```
(no need to change `ARCHITECTURES`, however, because that platform is no longer supported, so we can keep the "?" in there.)
Thanks!

Reply:
Will the buildbot be able to pick it up if you keep the "?" in there?

Reply:
AFAIK no.

Reply:
Then it should be removed, otherwise we won't get the x86 package on x86_gcc2?

Reply:
It will indeed not build it when not marked stable (i.e. without any prefix).

Reply:
What I'm suggesting is:
```
ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
SECONDARY_ARCHITECTURES="x86"
```
With these setting the build servers **will** create cutemarked_**x86** for x86_gcc2.

(When I said we could keep the "?" for x86 it was for `ARCHITECTURES`, not for `SECONDARY_ARCHITECTURES`. The purpose of my comment was to make sure we would not mark the recipe as tested on x86 **primary**.)